lib/modules/manager/pip-compile/extract.ts

```diff
@@ -130,6 +137,20 @@ export async function extractAllPackageFiles(
         config,
       );
       if (packageFileContent) {
+        for (const dep of packageFileContent.deps) {
+          const lockedVersion = lockedDeps?.find(
+            (lockedDep) =>
+              normalizeDepName(lockedDep.depName!) ===
+              normalizeDepName(dep.depName!),
+          )?.currentVersion;
+          if (lockedVersion) {
+            dep.lockedVersion = lockedVersion;
+          } else {
+            logger.warn(
+              `pip-compile: \`${dep.depName}\` not found in lock file \`${fileMatch}\``,
```

Comment:
```suggestion
              { depName: dep.depName, lockFile: fileMatch },
              `pip-compile: dependency not found in lock file`,
```

Avoid string interpolation in warning messages